### ISSUE TITLE
Getting lots of sRGB errors

### ISSUE THREAD

**Issue by SilentGlasses** [devreldragon/data-driven-dave#25]
Launching the games outputs lots of these errors:

```
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: cHRM chunk does not match sRGB
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: cHRM chunk does not match sRGB
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: cHRM chunk does not match sRGB
libpng warning: iCCP: known incorrect sRGB profile
```

Not sure if this is expected behavior but just wanted to bring this up.

**Comment by devreldragon** [comment]
Acknowledged. This was an issue inherited from the original repo. We'll have to look into the resolution. Thanks.

**Comment by devreldragon** [comment]
@SilentGlasses This issue was resolved with PR #26. If you refresh your local code, you will need to rerun the `pip3 install -r requirements.txt` command, as we have updated a number of things in the code including the New Relic agent version.
